.vscode/launch.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I do not use vscode myself so I have no way of telling if this is good or bad :D

Reply:
![image](https://user-images.githubusercontent.com/3540977/123660728-d1346f80-d82b-11eb-9ef6-312abf780287.png)

Reply:
:) well the comment in the JSON cannot be correct though.. syntactically incorrect...

Reply:
I can take them out, vscode itself put them there.

Reply:
yea i think that would be cleaner - this is only replicating what is already in the package.json

Reply:
I've taken it out, wasn't sure about having editor-specific config in the repo to be honest.